PyJobTransforms.<module> 2024-11-25 23:07:30,128 INFO logging set in /cvmfs/atlas-nightlies.cern.ch/repo/sw/main_Athena_x86_64-el9-gcc14-opt/2024-11-25T2101/Athena/25.0.23/InstallArea/x86_64-el9-gcc14-opt/bin/Reco_tf.py
PyJobTransforms.main 2024-11-25 23:07:30,357 INFO This is /cvmfs/atlas-nightlies.cern.ch/repo/sw/main_Athena_x86_64-el9-gcc14-opt/2024-11-25T2101/Athena/25.0.23/InstallArea/x86_64-el9-gcc14-opt/bin/Reco_tf.py
PyJobTransforms.transform.parseCmdLineArgs 2024-11-25 23:07:30,384 INFO Transform command line was: '/cvmfs/atlas-nightlies.cern.ch/repo/sw/main_Athena_x86_64-el9-gcc14-opt/2024-11-25T2101/Athena/25.0.23/InstallArea/x86_64-el9-gcc14-opt/bin/Reco_tf.py' '--CA' '--preInclude' 'InDetConfig.ConfigurationHelpers.OnlyTrackingPreInclude' '--inputRDOFile' '/cvmfs/atlas-nightlies.cern.ch/repo/data/data-art/PhaseIIUpgrade/RDO/ATLAS-P2-RUN4-01-01-00/mc21_14TeV.601229.PhPy8EG_A14_ttbar_hdamp258p75_SingleLep.recon.RDO.e8481_s4038_r14365/RDO.32520484._000001.pool.root.1' '--outputAODFile' 'AOD.test.root' '--steering' 'doRAWtoALL' '--postInclude' 'GeoModelConversion.GeoModelConversionConfig.GeoModelConversionAlgCfg' '--maxEvents' '1' 'Reco_tf.py' '--CA' '--preInclude' 'InDetConfig.ConfigurationHelpers.OnlyTrackingPreInclude' '--inputRDOFile' '/cvmfs/atlas-nightlies.cern.ch/repo/data/data-art/PhaseIIUpgrade/RDO/ATLAS-P2-RUN4-01-01-00/mc21_14TeV.601229.PhPy8EG_A14_ttbar_hdamp258p75_SingleLep.recon.RDO.e8481_s4038_r14365/RDO.32520484._000001.pool.root.1' '--outputAODFile' 'AOD.test.root' '--steering' 'doRAWtoALL' '--postInclude' 'GeoModelConversion.GeoModelConversionConfig.GeoModelConversionAlgCfg' '--maxEvents' '1'
PyJobTransforms.trfArgClasses.__call__ 2024-11-25 23:07:30,385 CRITICAL Got this exception raised when calling object factory: Failed to convert substep value Reco_tf.py to int (Error code 4)
PyJobTransforms.transform.parseCmdLineArgs 2024-11-25 23:07:30,385 CRITICAL Argument parsing failure: Failed to convert substep value Reco_tf.py to int (Error code 4)
